zkevm-circuits/src/evm_circuit.rs

```diff
@@ -252,7 +252,13 @@ impl<F: Field> SubCircuitConfig<F> for EvmCircuitConfig<F> {
     }
 }
 
-type AssignedChunkContextCell<F> = (AssignedCell<F, F>, AssignedCell<F, F>, AssignedCell<F, F>);
```

Comment:
Let's add a comment here to see clearly what each of this cells is
`/// chunk_index, chunk_index_next, total_chunk, initial_rwc, end_rwc `

Reply:
Done :)